rustls/src/msgs/handshake.rs

```diff
@@ -2252,3 +2253,125 @@ impl HandshakeMessagePayload {
         }
     }
 }
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct HpkeSymmetricCipherSuite {
+    pub hpke_kdf_id: HpkeKdf,
+    pub hpke_aead_id: HpkeAead,
+}
+
+impl Default for HpkeSymmetricCipherSuite {
```

Comment:
Probably sort this after the `Codec` impl?

Reply:
Done.